[Safe CPP] Address Warnings in LegacyRenderSVGResource

stwrt opened the pull request:
#### 0e40efaed0bee5249a6b0f21f2e0ad332f107e01
<pre>
[Safe CPP] Address Warnings in LegacyRenderSVGResource
<a href="https://bugs.webkit.org/show_bug.cgi?id=294178">https://bugs.webkit.org/show_bug.cgi?id=294178</a>
<a href="https://rdar.apple.com/problem/152800490">rdar://problem/152800490</a>

Reviewed by Chris Dumez.

Address safer cpp warnings in LegacyRenderSVGResource.

 * Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceClipperInlines.h:
 (WebCore::LegacyRenderSVGResourceClipper::clipPathUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilterInlines.h:
 (WebCore::LegacyRenderSVGResourceFilter::filterUnits const):
 (WebCore::LegacyRenderSVGResourceFilter::primitiveUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarker.cpp:
 (WebCore::LegacyRenderSVGResourceMarker::angle const):
 (WebCore::LegacyRenderSVGResourceMarker::markerTransformation const):
 (WebCore::LegacyRenderSVGResourceMarker::draw):
 (WebCore::LegacyRenderSVGResourceMarker::viewportTransform const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarker.h:
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarkerInlines.h:
 (WebCore::LegacyRenderSVGResourceMarker::protectedMarkerElement const):
 (WebCore::LegacyRenderSVGResourceMarker::markerUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMasker.cpp:
 (WebCore::LegacyRenderSVGResourceMasker::drawContentIntoMaskImage):
 (WebCore::LegacyRenderSVGResourceMasker::drawContentIntoContext):
 (WebCore::LegacyRenderSVGResourceMasker::calculateMaskContentRepaintRect):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMaskerInlines.h:
 (WebCore::LegacyRenderSVGResourceMasker::maskUnits const):
 (WebCore::LegacyRenderSVGResourceMasker::maskContentUnits const):

Canonical link: <a href="https://commits.webkit.org/295976@main">https://commits.webkit.org/295976@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/97c16e31299284d68f620fbec23b5a20f379a1ee

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/106825 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/26575 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/16973 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/112037 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/57422 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/108863 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/27245 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/35077 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/81119 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/109828 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/21629 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/96397 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/61460 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/21069 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/14505 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/56876 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/90970 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/14533 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/114996 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/33964 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/25055 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/90181 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/34327 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/92628 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/89891 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/22933 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/34828 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/12643 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/29648 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/33887 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/39302 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/33634 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/36987 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/35233 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/008fb768981db16fd316a98f4ba65b6e08400fac. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/46475

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/67aeeb0e6fcdc964eb5fc520fae0ed93f1a3b543. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/46475

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/97c16e31299284d68f620fbec23b5a20f379a1ee. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/46475

stwrt committed:
[Safe CPP] Address Warnings in LegacyRenderSVGResource
https://bugs.webkit.org/show_bug.cgi?id=294178
rdar://problem/152800490

Reviewed by Chris Dumez.

Address safer cpp warnings in LegacyRenderSVGResource.

 * Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceClipperInlines.h:
 (WebCore::LegacyRenderSVGResourceClipper::clipPathUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilterInlines.h:
 (WebCore::LegacyRenderSVGResourceFilter::filterUnits const):
 (WebCore::LegacyRenderSVGResourceFilter::primitiveUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarker.cpp:
 (WebCore::LegacyRenderSVGResourceMarker::angle const):
 (WebCore::LegacyRenderSVGResourceMarker::markerTransformation const):
 (WebCore::LegacyRenderSVGResourceMarker::draw):
 (WebCore::LegacyRenderSVGResourceMarker::viewportTransform const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarker.h:
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMarkerInlines.h:
 (WebCore::LegacyRenderSVGResourceMarker::protectedMarkerElement const):
 (WebCore::LegacyRenderSVGResourceMarker::markerUnits const):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMasker.cpp:
 (WebCore::LegacyRenderSVGResourceMasker::drawContentIntoMaskImage):
 (WebCore::LegacyRenderSVGResourceMasker::drawContentIntoContext):
 (WebCore::LegacyRenderSVGResourceMasker::calculateMaskContentRepaintRect):
 * Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMaskerInlines.h:
 (WebCore::LegacyRenderSVGResourceMasker::maskUnits const):
 (WebCore::LegacyRenderSVGResourceMasker::maskContentUnits const):

Canonical link: https://commits.webkit.org/295976@main
